dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -542,123 +542,79 @@ def wavepostsbs(self):
 
-        dep_dict = {'type': 'metatask', 'name': f'{self.run}_fcst_mem#member#'}
-        deps.append(rocoto.add_dependency(dep_dict))
-        dependencies = rocoto.create_dependency(dep_condition='or', dep=deps)
+        resources = self.get_resource('wave_stat')
 
-        wave_post_bndpnt_bull_envars = self.envars.copy()
-        postenvar_dict = {'ENSMEM': '#member#',
-                          'MEMDIR': 'mem#member#',
-                          }
-        for key, value in postenvar_dict.items():
-            wave_post_bndpnt_bull_envars.append(rocoto.create_envar(name=key, value=str(value)))
+        # Adjust walltime based on the largest group
+        largest_group = max([len(grp.split(',')) for grp in fhr_var_dict['fhr_list'].split(' ')])
+        resources['walltime'] = Tasks.multiply_HMS(resources['walltime'], largest_group)
 
-        resources = self.get_resource('wavepostbndpntbll')
-        task_name = f'{self.run}_wave_post_bndpnt_bull_mem#member#'
+        task_name = f'{self.run}_wave_stat_#fhr_label#'
         task_dict = {'task_name': task_name,
                      'resources': resources,
                      'dependency': dependencies,
-                     'envars': wave_post_bndpnt_bull_envars,
+                     'envars': wave_stat_envars,
                      'cycledef': self.run,
-                     'command': f'{self.HOMEgfs}/dev/jobs/wavepostbndpntbll.sh',
+                     'command': f'{self.HOMEgfs}/dev/jobs/wave_stat.sh',
                      'job_name': f'{self.pslot}_{task_name}_@H',
                      'log': f'{self.rotdir}/logs/@Y@m@d@H/{task_name}.log',
                      'maxtries': '&MAXTRIES;'
                      }
 
-        member_var_dict = {'member': ' '.join([str(mem).zfill(3) for mem in range(0, self.nmem + 1)])}
-        member_metatask_dict = {'task_name': f'{self.run}_wave_post_bndpnt_bull',
-                                'task_dict': task_dict,
-                                'var_dict': member_var_dict
-                                }
+        fhr_metatask_dict = {'task_name': f'gefs_wave_stat',
+                             'task_dict': task_dict,
+                             'var_dict': fhr_var_dict}
 
-        task = rocoto.create_task(member_metatask_dict)
+        task = rocoto.create_task(fhr_metatask_dict)
 
         return task
 
-    def wavepostpnt(self):
+    def wave_stat_pnt(self):
+
         deps = []
-        dep_dict = {'type': 'metatask', 'name': f'{self.run}_fcst_mem#member#'}
+        metatask = f'{self.run}_wave_stat'
+        dep_dict = {'type': 'metatask', 'name': metatask}
         deps.append(rocoto.add_dependency(dep_dict))
-        if self.options['do_wave_bnd']:
-            dep_dict = {'type': 'task', 'name': f'{self.run}_wave_post_bndpnt_bull_mem#member#'}
-            deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep_condition='and', dep=deps)
```

Comment:
```suggestion
        dependencies = rocoto.create_dependency(dep=deps)
```
There is only one dependency, `and` is redundant